components/badge/script.js

```diff
@@ -1,11 +1,10 @@
-const inputOne=document.querySelector("#btn-ide");
-
-function myFunction(){
-    const userInput=document.querySelector("#toggle1");
-    if(userInput.style.visibility === "visible"){
-        userInput.style.visibility="hidden";
+const btnInput=document.querySelector("#btn-input");
+const hideBadge=()=>{
+    const badge=document.querySelector("#badge-hide");
```

Comment:
you can create this reference at the starting of the document, it's a better practice I guess.